oxd-server/src/test/resources/testng.xml

```diff
@@ -5,6 +5,8 @@
     <parameter name="host" value="http://localhost"/>
     <parameter name="opHost" value="https://${test.server.name}"/>
     <parameter name="redirectUrl" value="https://client.example.com/cb"/>
+    <parameter name="redirectUrls" value="https://client.example.com/cb/home1,https://client.example.com/cb/home2"/>
```

Comment:
in connect parameters are passed with space delimiter, lets use it here too (not comma)
```
 <parameter name="redirectUrls" value="https://client.example.com/cb/home1 https://client.example.com/cb/home2"/>
```

Reply:
Changes done.